js/passage.js

```diff
@@ -1,4 +1,127 @@
-const passage = Object.freeze([
+export class Passage {
+  constructor(verses) {
+    this._verses = verses;
+  }
+
+  metadata() {
+    let firstVerse = this._verses[0];
+    let lastVerse = this._verses[this._verses.length - 1]
+    let bookChapter = firstVerse.book + ' ' + firstVerse.chapter + ':';
+    if (this._verses.length > 1) {
+      if (firstVerse.chapter === lastVerse.chapter) {
```

Comment:
I think we should address it (or add it as an issue), otherwise we'll forget about it if / when this is used for other cases in the future.